src/App.js

```diff
@@ -19,18 +19,20 @@ export default () => {
       )
     );
 
+  const onSubmit = useCallback(
+    text => setTodos(todos => [{ text, complete: false }, ...todos]),
```

Comment:
I like this solution. I hadn't thought about passing a function to `setTodos` to get the current state.